src/components/Register/RegisterForm.js

```diff
@@ -10,7 +10,8 @@ class RegisterForm extends Component {
             username: '',
             password: '',
             name: '',
-            email: ''
+            email: '',
+            toProfile: false
```

Comment:
This can be removed, again from another attempt to go to the profile page using redirects